### PR TITLE
Add wilderness cannon spots

### PR DESCRIPTION
**Pull request by Honestidade**
Update cannon spots to add the Wilderness slayer spots as outlined in the Oblivion Wilderness Slayer Points guide.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonSpots.java [hunk 1]
```diff
@@ -6,10 +6,10 @@
  * modification, are permitted provided that the following conditions are met:
  *
  * 1. Redistributions of source code must retain the above copyright notice, this
- *    list of conditions and the following disclaimer.
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonSpots.java]: Revert these indentation changes to the license header.

**Comment by raiyni** [comment]
Can you provide screenshots of these locations?

**Comment by Honestidade** [comment]
> 
> 
> Can you provide screenshots of these locations?

https://i.imgur.com/mBz52w9.png
https://i.imgur.com/ywe8est.png
https://i.imgur.com/85xafck.png
https://i.imgur.com/y4GZPdO.png
https://i.imgur.com/ReACn7m.png
https://i.imgur.com/XYuWfj4.png
https://i.imgur.com/RIwX0kV.png
https://i.imgur.com/1alqH3J.png
https://i.imgur.com/XYG9nB6.png
https://i.imgur.com/jTtSgKl.png
https://i.imgur.com/eBI9018.png
https://i.imgur.com/ZOwEX7j.png
https://i.imgur.com/uCCH1zx.png
https://i.imgur.com/MeWGK3J.png

**Comment by Honestidade** [comment]
Should be fixed now, no clue how that even got edited in the first place.

File: runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonSpots.java [hunk 2]
```diff
@@ -36,7 +36,7 @@
 	BLOODVELDS(new WorldPoint(2439, 9821, 0), new WorldPoint(2448, 9821, 0), new WorldPoint(2472, 9833, 0), new WorldPoint(2453, 9817, 0)),
 	FIRE_GIANTS(new WorldPoint(2393, 9782, 0), new WorldPoint(2412, 9776, 0), new WorldPoint(2401, 9780, 0)),
 	ABBERANT_SPECTRES(new WorldPoint(2456, 9791, 0)),
-	HELLHOUNDS(new WorldPoint(2431, 9776, 0), new WorldPoint(2413, 9786, 0), new WorldPoint(2783, 9686, 0)),
+	HELLHOUNDS(new WorldPoint(2431, 9776, 0), new WorldPoint(2413, 9786, 0), new WorldPoint(2783, 9686, 0), new WorldPoint(3198, 10071, 0)),
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonSpots.java]: Why was this worldpoint removed?

File: runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonSpots.java [hunk 3]
```diff
@@ -47,12 +47,25 @@
 	DARK_BEAST(new WorldPoint(1992, 4655, 0)),
 	DUST_DEVIL(new WorldPoint(3218, 9366, 0)),
 	KALPHITE(new WorldPoint(3307, 9528, 0)),
-	LESSER_DEMON(new WorldPoint(2838, 9559, 0)),
+	LESSER_DEMON(new WorldPoint(2838, 9559, 0), new WorldPoint(3163, 10114, 0)),
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonSpots.java]: Same here

File: runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonSpots.java (hunk 2 above)
Comment (MagicfTail) [reply runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonSpots.java]: It isn't removing points, it's just adding additional locations

**Pull request merged**